include/grpc/grpc.h

```diff
@@ -201,22 +193,15 @@ typedef struct grpc_metadata {
 } grpc_metadata;
 
 typedef enum grpc_completion_type {
-  GRPC_QUEUE_SHUTDOWN,       /* Shutting down */
-  GRPC_OP_COMPLETE,          /* operation completion */
-  GRPC_SERVER_SHUTDOWN,      /* The server has finished shutting down */
```

Comment:
Cold you please update the doc-comment for `grpc_server_shutdown_and_notify`, then (line 505 in the original file, 485 in the edited)?